gnomad_qc/v4/create_release/create_combined_faf_release_ht.py

```diff
@@ -458,6 +458,10 @@ def main(args):
                 cochran_mantel_haenszel_test=res.cmh_ht.ht()[
                     ht.key
                 ].cochran_mantel_haenszel_test,
+                data_type=hl.case()
+                .when(~hl.is_defined(ht.genomes_freq), "exomes")
+                .when(~hl.is_defined(ht.exomes_freq), "genomes")
+                .default("both"),
```

Comment:
@KoalaQin, faf is not always defined even when freq is (variant is see only in remaining or a bottleneck population) so we'll want an extra case statement for `both` and then I think the default should be missing.

Reply:
This statement ^^^ is incorrect is defined but 0 for both rare variants and groups with AC =0.

Reply:
I just changed the code and tested it on the test gene, could you both check the naming and struct? @mike-w-wilson @ch-kr